README.md

```diff
@@ -166,6 +170,7 @@ location_id                  | Text | Primary key
 location_type                | Text |
 longitude                    | Text |
 name                         | Text |
+parent                       | Text |
```

Comment:
Yes, it's a resource URI. I kept the name 'parent' because that's what appears in the JSON if you read locations using the API, but I don't mind changing it.